src/components/general/NavigationDrawer/components/Grouping.tsx

```diff
@@ -37,7 +34,9 @@ const Grouping: FC<NavigationComponentProps> = ({ navigationItem, onChange, isCo
                     className={styles.toggleOpenContainer}
                     onClick={() => onChange && onChange(id, true, false)}
                 >
-                    {!isCollapsed && <DropdownArrow cursor="pointer" isOpen={isOpen || false} />}
+                    {!isCollapsed && navigationChildren && navigationChildren.length > 0 && (
+                        <DropdownArrow cursor="pointer" isOpen={isOpen || false} />
```

Comment:
Maybe add navigationChildren in the dependency array? If the structure gets updated/changed while the component is mounted, we also want the DropdownAroow to act accordingly

Reply:
Fixed